coordinator/persistence-cassandra-5.0/README.md

```diff
@@ -0,0 +1,28 @@
+# Persistence Cassandra 5.0
+
+This module represents the implementation of the [persistence-api](../persistence-api) for the Cassandra `5.0.x` version.
+
+Note: the initial implementation is based on an experimental fork of Cassandra implementation and may include code that has not yet been accepted into the Apache Cassandra project.
+
+## Cassandra version update
+
+The current Cassandra version this module depends on is `4.1.2`.
```

Comment:
Fix - should be 4.0.7-<commit-sha> with reference to https://github.com/datastax/cassandra/tree/vsearch